pkg/proxy/types.go

```diff
@@ -42,3 +43,33 @@ type ServicePortName struct {
 func (spn ServicePortName) String() string {
 	return fmt.Sprintf("%s:%s", spn.NamespacedName.String(), spn.Port)
 }
+
+// ServicePort is an interface which abstracts a service info.
```

Comment:
Same as below. WDYT about "ServicePort is an interface which abstracts information about a service"

Reply:
Great! It's fixed now.